### ISSUE TITLE
DDL sanitation fails when identifier is quoted.

### ISSUE THREAD

**Issue by morgen-peschke** [opt-tech/redshift-fake-driver#21]
I ran into this behavior while testing some generated SQL, it looks like the regex for `DISTKEY` and `SORTKEY` need to be tweaked a bit.

Failing case
=========

Input
-----
```
println(DDLParser.sanitize("""CREATE TEMPORARY TABLE test_table
                             |(
                             |  "test_identifier" INT NOT NULL,
                             |)
                             |DISTKEY("test_identifier")
                             |SORTKEY("test_identifier");""".stripMargin))
```

Output
------
```
CREATE TEMPORARY TABLE test_table
(
  "test_identifier" INT NOT NULL,
)
DISTKEY("test_identifier)
SORTKEY("test_identifier);
```

Passing Case
==========
Input
-----
```
println(DDLParser.sanitize("""CREATE TEMPORARY TABLE test_table
                             |(
                             |  test_identifier INT NOT NULL,
                             |)
                             |DISTKEY(test_identifier)
                             |SORTKEY(test_identifier);""".stripMargin))
```

Output
-------
```
CREATE TEMPORARY TABLE test_table
(
  test_identifier INT NOT NULL,
);
```

**Comment by ocadaruma** [comment]
Thank you for reporting the issue.
I will check by this week.

**Comment by ocadaruma** [comment]
@morgen-peschke FIxed by #24 . Please check it.

**Comment by morgen-peschke** [comment]
Looks good to me
